src/components/MediaItem/MediaItemTitle.tsx

```diff
@@ -73,12 +82,17 @@ export default function MediaItemTitle({ i, data, type, parentRef }: Props) {
       window.removeEventListener('resize', debouncedRestartAnimation)
       debouncedRestartAnimation.cancel()
     }
-  }, [controls, parentRef])
+  }, [controls, parentRef, data.user_id])
 
   return (
-    <h3 className='mb-2 flex sm:text-xl'>
+    <h3 className={clsx('mb-2 flex sm:text-xl', isLoggedIn && 'gradient-plex')}>
       <span className='mr-1.5 inline-flex items-baseline gap-1' ref={numberRef}>
-        <span className={clsx('font-bold', topColors[i] || 'text-white')}>
+        <span
+          className={clsx(
+            'font-bold',
+            !isLoggedIn && (topColors[i] || 'text-white'),
```

Comment:
I would preserve the placement color irregardless of `isLoggedIn`.
Looks a bit weird when the logged in user is in 2nd place and the gradient-plex makes the whole row look very similar to 1st (gold).